bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/internal/DialogProcessor.java

```diff
@@ -184,12 +186,17 @@ protected void say(String text) {
                 }
             }
             if (null == voice) {
-                throw new TTSException("Unable to find a suitable voice");
+                throw new TTSException("Unable to find a voice");
```

Comment:
why did you change this? It seems unrelated to this PR and the message is not correct anymore as the system probably was able to find a voice, but not one of the right locale.